src/test/java/api/support/fixtures/ScheduledNoticeProcessingTimerClient.java

```diff
@@ -0,0 +1,22 @@
+package api.support.fixtures;
+
+import static api.support.RestAssuredClient.post;
+
+import java.net.URL;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+
+import api.support.APITestContext;
+import io.vertx.core.json.JsonObject;
+
+public class ScheduledNoticeProcessingTimerClient {
+
```

Comment:
Can these blank lines be removed. Are they present for readability?

Reply:
Done